test/features/page_objects/main.js

```diff
@@ -1,4 +1,4 @@
-let e = {};
+const e = {};
```

Comment:
This object is being changed (extended with additional properties) in the next 2 lines, so it looks like it should be `let` instead of `const`